source/particle/world.cc

```diff
@@ -163,26 +161,39 @@ namespace aspect
     {
       signals.post_set_initial_state.connect(std_cxx11::bind(&World<dim>::setup_initial_state,
                                                              std_cxx11::ref(*this)));
+
       signals.pre_refinement_store_user_data.connect(std_cxx11::bind(&World<dim>::register_store_callback_function,
                                                                      std_cxx11::ref(*this),
+                                                                     false,
                                                                      std_cxx11::_1));
+      signals.pre_serialization_store_user_data.connect(std_cxx11::bind(&World<dim>::register_store_callback_function,
```

Comment:
this is pre-existing, but I think the function you are attaching here is poorly named: "register_*" functions are ones that *connect a function X to a signal S*, but you actually name the function X itself this way. Maybe rename these functions `store_callback_function` and `load_callback_function`?